target/linux/ath79/dts/qca9533_comfast_cf-e110n.dts

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-e110n", "qca,qca9533";
+	model = "COMFAST CF-E110N";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &wan;
+		led-failsafe = &wan;
+		led-upgrade = &rssilow;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wan: wan {
+			label = "cf-e110n:green:wan";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
```

Comment:
You're using GPIOs 0,2,3 in your DTS, in your ar71xx implementation you've had `ath79_gpio_function_enable(AR934X_GPIO_FUNC_JTAG_DISABLE);` but I don't see the `jtag_disable_pins` pinctrl being used anywhere.

Reply:
You're right. I had it in some previous version but then I started to add, remove and change stuff to see if I could catch what was going wrong with the GPIO pins, so I must have removed it.

```
keys {
		compatible = "gpio-keys-polled";
		poll-interval = <20>;

		pinctrl-0 = <&jtag_disable_pins>;
		
		reset {
			label = "Reset button";
			linux,code = <KEY_RESTART>;
			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
			debounce-interval = <60>;
		};
	};
```